.rustfmt.toml

```diff
@@ -0,0 +1,31 @@
+# Basic
+edition = "2021"
+max_width = 100
+use_small_heuristics = "Max"
+
+# Imports
+imports_granularity = "Preserve"
+reorder_imports = false
+reorder_modules = false
+
+# Consistency
+newline_style = "Unix"
+
+# Misc
+chain_width = 80
+spaces_around_ranges = false
+reorder_impl_items = false
+match_arm_leading_pipes = "Preserve"
+match_arm_blocks = false
+match_block_trailing_comma = true
+trailing_comma = "Vertical"
+# trailing_semicolon = false
+# use_field_init_shorthand = true
+
+# where_single_line = true  # does not work on fn
+# brace_style = "AlwaysNextLine"  # does not work on method fn
+
+# Format comments
+comment_width = 100
+wrap_comments = true
```

Comment:
Your git GUI has seemingly pulled recent changes from master into its own commit.  I'd expect git handles this fine, likely disapears once rebased onto master, but it's odd behavior that indicates issues with the git GUI.

Did you make any other changes to this PR since 25 April?

Reply:
Yeah, I messed up...Pushed some changes directly to master instead of this branch and then pulled them.